Script/AtomicEditor/hostExtensions/HostExtensionServices.ts

```diff
@@ -296,69 +296,69 @@ export class UIServicesProvider extends ServicesProvider<Editor.HostExtensions.U
      */
     menuItemClicked(refid: string): boolean {
         // run through and find any services that can handle this.
-        let holdResult = false;
-        this.registeredServices.forEach((service) => {
+        return this.registeredServices.some((service) => {
             try {
```

Comment:
I sometimes think of poor TorqueScript and it's lack of try/catch, no problem here.